dev/prepare_env.sh

```diff
@@ -3,11 +3,11 @@
 # Anaconda needs to be installed upfront
 conda update -n base -c defaults conda
 conda create -y --name tvb-widgets python=3.8 nomkl numba scipy numpy
-conda install -y --name tvb-widgets -c conda-forge jupyterlab tvb-library tvb-data ipywidgets==7.7.1
+conda install -y --name tvb-widgets -c conda-forge jupyterlab tvb-library tvb-data ipywidgets==7.7.2
 conda install -y --name tvb-widgets -c conda-forge pythreejs pyvista colorcet
 conda install -y --name tvb-widgets ipywidgets
 
 pip install --upgrade pip
-pip install ebrains_drive mne
+pip install ebrains_drive mne==0.24.1
```

Comment:
for mne we expect >=1

Reply:
yes, sorry, I think this was the results of a bad merge, I will fix it

Reply:
no big deal at all. This file seems just a developer documentation. it does not seem to be used